lib/twilio/device.ts

```diff
@@ -380,6 +380,11 @@ class Device extends EventEmitter {
    */
   private _log: Log = new Log('Device');
 
+  /**
+   * The internal promise created when calling {@link Device.makeCall}.
+   */
+  private _makeCallPromise: Promise<any> = Promise.resolve();
```

Comment:
This needs to be optional and should not have a default value. The value should only be present when there's a call, and shouldn't be present when there's no call.
```ts
private _makeCallPromise: Promise<any> | null = null;
```